uis/layover/app/partials/delete-cache.html

```diff
@@ -0,0 +1,33 @@
+<!--
+
+    Copyright (C) 2011-2017 Red Hat, Inc. (https://github.com/Commonjava/indy)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<div class="container-fluid" ng-controller="CacheCtl">
+  <div class="control-panel">
+    <div class="cp-row">
+      Input the URL of cached file to delete (for group and remote repository storage):
```

Comment:
Is this meant to be a full URL, or only a URL path like `/api/content/maven/group/central/org/foo/bar/1/bar-1.pom`? Is there any validation limiting this, other than the bounds of what the browser will allow? Can I delete something in another Indy instance from the one running on my localhost, assuming they use the same auth info?

I wonder if we need to add some validation in the javascript to ensure people can't put in random URLs? Or maybe it's already there?

Also, it might be good to put in some guidance for the user in this form, as to its intention and whatever validation we do.

Reply:
I tested and the js seems smart enough that both works, e.g., a full url or /api/content/maven/group/central/org/foo/bar/1/bar-1.pom
If inputing something bad, like /abc/xxx, no js validation yet, but the result will show errors from Indy. 
Yes I think you can delete sth from other servers if the auth allows. 
I will add some guidance in the form.